examples/matRad_example18_CT_sCT_DVH_difference_photons.m

```diff
@@ -0,0 +1,254 @@
+%% Example: Photon Treatment Plan
+%
+% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+% Copyright 2017 the matRad development team. 
+% 
+% This file is part of the matRad project. It is subject to the license 
+% terms in the LICENSE file found in the top-level directory of this 
+% distribution and at https://github.com/e0404/matRad/LICENSE.md. No part 
+% of the matRad project, including this file, may be copied, modified, 
+% propagated, or distributed except according to the terms contained in the 
+% LICENSE file.
+%
+% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+
+%% In this example, we will show:
+% (i) how to load DICOM patient data into matRad and modify dosimetric
+% objectives and constraints;
+% (ii) how to set up a photon dose calculation and optimization based on the real CT;
+% (iii) how to translate the optimized plan to be applied to a synthetic CT image; and
+% (iv) how to calculate the DVH differences between the plans.
+
+%% Patient Data Import from DICOM  
+% Start with a clean MATLAB environment, then import the real CT data of a patient.  
+% For this purpose, LIVER phantom data has been adapted and saved as a DICOM file with a structure set.  
+% Functions for importing DICOM files will search for .dcm files in the directory  
+% and automatically recognize 3D volumes and structure files.  
+% The DICOM files will then be read into the workspace as 'ct' and 'cst',  
+% representing the CT images and the structure set, respectively.  
+% Ensure that the matRad root directory, along with all its subdirectories,  
+% is added to the MATLAB search path.  
+
+clear;
+matRad_cfg = matRad_rc; %If this throws an error, run it from the parent directory first to set the paths
+patDir= "matRad/phantoms/synthetic_ct/"; % If you want to export your data further, use "userdata" folder
+name="pat001";
+patDir_real_CT= fullfile(patDir,name,"real_CT");
+imp_real_CT = matRad_DicomImporter(patDir_real_CT);
+matRad_importDicom(imp_real_CT);
+% Review the exported file and automatically identified  
+% optimization objectives and constraints. 
+matRadGUI;
+
+%% Modifying Plan Optimization Objectives  
+% The constraints for all defined volumes of interest (VOIs) are stored in the cst cell.  
+% When importing DICOM structure files, matRad uses matRad_createCst() to generate this cell.  
+% Default regular expressions are used to define target objectives. However,  
+% additional constraints for organs at risk (OARs) need to be added for the plan.  
+% This can be done by directly modifying the cst cell.  
+% If different volumes with varying optimization objectives need to be loaded,  
+% refer to the documentation for matRad_createCst() and choose suitable options  
+% to create a custom implementation. In this case, the plan will be modified directly here.  
+
+for i = 1:size(cst, 1)
```

Comment:
Can you slightly rewrite to point out a bit more that this for loop is only one of the methods to assign objectives to the current structure (e.g. if you do not want to check manually where they are stored or you analyze multiple synCT/CT pairs where the naming is not always perfectly consistent?